docs/includes/installing-fastlane.md

```diff
@@ -4,12 +4,41 @@ Install the latest Xcode command line tools:
 xcode-select --install
 ```
 
-Install _fastlane_ using
+### Install with Bundler (recommended)
+The recommended way to install _fastlane_ is with [Bundler](https://bundler.io/) by adding _fastlane_ to your `Gemfile`. `bundler` provides a consistent execution environment for your _fastlane_ project to run in.
```

Comment:
"adding" indicates you already have one. For most fastlane users that won't be true, right?